### PR TITLE
VIH-7853 Unlabelled forms/ungrouped buttons in Judge Waiting Room

### PR DESCRIPTION
**Pull request by ZJianWang**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7853

### Change description ###

Labelled forms and grouped buttons in Judge Waiting Room

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/chat-input-box/chat-input-box.component.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <div class="chat-input-group">
     <div [ngClass]="isInputInvalid ? '' : 'govuk-form-group  govuk-!-margin-bottom-2'"
          class="chat-input-text-button-group govuk-!-margin-left-2">
+         <label for="new-message-box"></label>
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/chat-input-box/chat-input-box.component.html:4]: There is no content for this label. We need to have an English and a Welsh content.

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/hearing-layout.component.html [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+    [value]="layout"
+    (change)="emitSelected()"
+    class="govuk-radios__input"
+    id="layout-radio-button-{{layout}}"
+    name="layout"
+    type="radio"
+    attr.aria-labelledby="{{headerLayoutTitlePrefix}}{{layout}}"
+    attr.aria-describedby="{{headerLayoutDescriptionPrefix}}{{layout}}"
+  />
+  <label class="govuk-label govuk-radios__label"></label>
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/hearing-layout.component.html:13]: We should remove this label. I know it makes them display wrong but we should handle that with CSS on the input rather that relying on an empty label.

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/chat-input-box/chat-input-box.component.html (hunk 1 above)
Comment (ZJianWang) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/chat-input-box/chat-input-box.component.html:4]: if we put some English and Welsh to the label means we need hide this element

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/hearing-layout.component.html [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
   <div class="layoutimg">
     <img [attr.alt]="'hearing-layout.layout-image-alt' | translate: {layout: getLayoutTitle()}" class="layout-image" [src]="getLayoutImagePath()" />
   </div>
   <div class="layouttxt">
     <div class="govuk-body f16">
-      <strong>{{ getLayoutTitle() }}</strong>
+      <strong id="{{showInput ? '' : 'accordian-header-'}}{{headerLayoutTitlePrefix}}{{layout}}">{{ getLayoutTitle() }}</strong>
       <span *ngIf="recommended" class="recommended-txt"> ({{'hearing-layout.recommended' | translate}})</span>
-      <br />{{ getLayoutDescription() }}
+      <br /><p id="{{showInput ? '' : 'accordian-header-'}}{{headerLayoutDescriptionPrefix}}{{layout}}">{{ getLayoutDescription() }}</p>
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/hearing-layout.component.html:23]: You can remove the <br /> after adding the <p> element. We should avoid using <br /> the majority of the time 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

**Review by robertbennett1998 — APPROVED** [review]
LGTM. Good job tidying the select-hearing-layout component :D

**Review by AWildBugAppeared — APPROVED** [review]

**Pull request merged**